lisa/wa.py

```diff
@@ -255,6 +255,13 @@ def _needs_params(col):
             for name, param in parameters[1:]
         )
 
+    @property
+    @memoized
+    def jobs(self):
+        outputs = list(discover_wa_outputs(self.path))
+        if len(outputs) > 1:
+            raise DeprecationWarning('Passing a directory containing multiple outputs of "wa run" to WAOutput is deprecated. Please, create one object per run.')
+        return [job for job in outputs[0].jobs if job.status == Status.OK]
```

Comment:
We should still use all of `outputs`, not just `[0]`. Maybe in time we can actually get rid of the others but for now it's only going to be deprecated, not removed